spec/models/document_model/html_title_spec.rb

```diff
@@ -0,0 +1,18 @@
+#require 'rails_helper'
```

Comment:
Pretty stuck on writing a test for this module :-/

Reply:
```
RSpec.describe DocumentModel::HtmlTitle do
  let(:some_class) { Class.new { extend DocumentModel::HtmlTitle } }

  describe "#html_title_qualifier" do
    let(:title) { 'A title of a document that is probably descriptive/unique' }
    let(:qualifier) { 'ark:/99999/1234567' }

    it "should be blank, no identifier rendered" do
     expect(some_class.html_title_qualifier(title, qualifier)).to eq("")
    end
  end
end

RSpec.describe DocumentModel::HtmlTitle do
  let(:some_class) { Class.new { include DocumentModel::HtmlTitle } }

  let(:foo_instance) { some_class.new }

  describe "#html_title_qualifier" do
    let(:title) { 'A title of a document that is probably descriptive/unique' }
    let(:qualifier) { 'ark:/99999/1234567' }

    it "should be blank, no identifier rendered" do
     expect(foo_instance.html_title_qualifier(title, qualifier)).to eq("")
    end
  end
```